CWE/events/east_turkestan.txt

```diff
@@ -89,6 +89,8 @@ country_event = {
 			neighbour = THIS
 		}
 		
+		NOT = { XIJ = { war_with = ETS } #Ili Rebellion }
```

Comment:
NOT = { XIJ = { war_with = ETS } #Ili Rebellion } will cause bugs, # has to be outside any brackets for each line.

NOT = { XIJ = { war_with = ETS } } #Ili Rebellion